role-management-commands/list-temproles.md

```diff
@@ -1,6 +1,15 @@
 # list-temproles
 
-<TODO: describe>
+This command allows you to view all active temporary roles and paused roles in the server. It will display the user,
+role, and the time remaining for each temporary/paused role.
+
+You can also specify a target to filter the results to only show the temporary roles of a user or the users of a role.
```

Comment:
This and the following lines are unnecessary due to the close proximity of the information

Reply:
![paveikslas](https://github.com/MoonlightCapital/MoonlightBot-docs/assets/19293406/981c56e3-c4aa-4359-9bb0-aec516ea5703)

timer emoji indicates a temprole, pause button emoji a pause-role